### ISSUE TITLE
axe-linter error failing complete pull request 

### ISSUE THREAD

**Issue by ShaktiMandal** [khalidelaggan/axe-linter#1]
I wouldn't call this is an issue but require information about axe-linter configuration. As per my current requirement, I would like to continue running the build even after having accessibility error from axe-linter. Thus, I can manually check the error and decide to fix it now or later. In current situation, build process is getting stopped whenever there is an accessibility issue. Therefore complete pull request is getting failed. Please let me know if this can be done. If yes, what would be the procedure to do so.
